api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -63,8 +63,14 @@ message LocalRateLimit {
   // Defaults to 0% of requests for safety.
   config.core.v3.RuntimeFractionalPercent filter_enforced = 5;
 
+  // Specifies a list of HTTP headers that should be added to each request that
+  // has been rate limited and is also forwarded upstream. This can only occur when the
+  // filter is enabled but not enforced.
+  repeated config.core.v3.HeaderValueOption request_headers_to_add = 10
```

Comment:
This probably needs a different name, maybe `request_headers_to_add_when_not_enforced`?